src/Cell.ts

```diff
@@ -12,6 +12,8 @@ export interface CellProperties extends HasColumn, HasItem, HasValue, RegistryMi
 	key: string;
 }
 
+export type CellInterface = WidgetBaseConstructor<CellProperties>;
```

Comment:
Like `BodyInterface` above.  Also not clear on why you can't use `typeof` something to generate this.  It doesn't feel like it is a complete type... all it is doing is filling in a generic, which you could do where you actually need to constrain to that.